mage/kubectl/lib.go

```diff
@@ -14,13 +14,23 @@ import (
 	mageutil "github.com/datastax/cass-operator/mage/util"
 )
 
-func GetKubeconfig() string {
+func GetKubeconfig(createDefault bool) string {
```

Comment:
not sure if jamming the logic into this function is the best idea or not, but it does save from having to lookup user dir in multiple places

Reply:
yeah I think this is fine